paymentsheet/src/main/java/com/stripe/android/lpmfoundations/paymentmethod/UiDefinitionFactory.kt

```diff
@@ -37,13 +41,16 @@ internal sealed interface UiDefinitionFactory {
                 private val cardAccountRangeRepositoryFactory: CardAccountRangeRepository.Factory,
                 private val paymentMethodCreateParams: PaymentMethodCreateParams? = null,
                 private val paymentMethodExtraParams: PaymentMethodExtraParams? = null,
+                private val linkConfigurationCoordinator: LinkConfigurationCoordinator? = null,
```

Comment:
I think we should remove these defaults (assuming it's only needed for tests).

Reply:
I think it's worth being explicit, and documenting why it's null/ and making the callback throw if it is called.